### PR TITLE
Implement "Add participants" page

### PR DESCRIPTION
**Pull request by doskachok**
Implemented only Equally split method

### CODE REVIEW

File: src/features/expenses/AddExpense/AddParticipants.tsx [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+  const group = useGetTransactionGroupByIdQuery(transaction?.groupId || '');
+
+  const [splitMethodIndex, setSplitMethodIndex] = useState<number>(0);
+
+  const handleBackAction = useCallback(() => {
+    navigate(-1);
+  }, [navigate]);
+
+  const resetPartialsAssignments = useCallback((splitIndex: number) => {
+    const newSplitMethod = splitMethodIndexToEnum(splitIndex);
```
Comment (Gamgeee) [inline src/features/expenses/AddExpense/AddParticipants.tsx]: Try to implement something like SplitMethod[splitIndex]

File: src/features/expenses/AddExpense/AddParticipants.tsx [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+          </ButtonBase>
+
+          <SplitMethodWrapper fullWidth jc="space-between">
+            <SplitMethodBorderWrapper>
+              <BorderShift flex={splitMethodIndex} />
+              <MovingBorder />
+              <BorderShift flex={3 - splitMethodIndex} />
+            </SplitMethodBorderWrapper>
+
+            <SplitMethodButton text={t("equally")} svgImg={<EquallyMethodSVG />} onClick={onSplitMethods[0]} />
```
Comment (Gamgeee) [inline src/features/expenses/AddExpense/AddParticipants.tsx]: Please try to use enum here

File: src/features/expenses/AddExpense/AddParticipants.tsx (hunk 1 above)
Comment (doskachok) [reply src/features/expenses/AddExpense/AddParticipants.tsx]: It was changed in 81508842d73fd52a080d5628b9689b50a2437d2d, please have a look.

File: src/features/expenses/AddExpense/AddParticipants.tsx (hunk 2 above)
Comment (doskachok) [reply src/features/expenses/AddExpense/AddParticipants.tsx]: Done in 81508842d73fd52a080d5628b9689b50a2437d2d

**Review by Gamgeee — APPROVED** [review]

**Pull request merged**